config/base.js

```diff
@@ -11,10 +11,9 @@ module.exports = {
   */
   extends: [
     'eslint:recommended',
-    // Ember suave includes a bunch of non-Ember-related rules that we want to
-    // use, so we include it here. The Ember-related ones should not have any
-    // effect on non-Ember code.
-    'plugin:ember-suave/recommended'
+    // Contains only recommended settings for custom rules defined in the
+    // eslint-plugin-ember addon
+    'plugin:ember/base'
```

Comment:
I think it would make more sense to just opt in to the rules we want. So don't bother extending their base at all, and don't turn off a bunch or rules below, just explicitly turn on the ones we want.